content/posts/0004-writing-buildpacks.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Building a Cloud Native Buildpack Isn’t as Hard as You Think
+date: "2021-02-16"
+slug: building-a-cloud-native-buildpack-isnt-as-hard-as-you-think
+author: swigmore
+---
+Maybe you’ve heard that many companies today are transitioning to, or have already transitioned to, a container-based platform like Kubernetes. With this movement comes the need to containerize your applications, new and old. Buildpacks provide a seamless mechanism for doing that, *without Dockerfiles.* [Paketo Buildpacks](https://paketo.io) is a community-driven project that provides Cloud Native Buildpacks implementations for the most popular languages. Buildpacks provide great out-of-box support for many languages, but what if you need a buildpack that doesn’t exist yet? As a community-driven initiative, it may feel like the onus is on you to create that buildpack. Where would you even begin? 🤯
```

Comment:
```suggestion
Maybe you’ve heard that many companies today are transitioning to, or have already transitioned to, a container-based platform like Kubernetes. With this movement comes the need to containerize your applications, new and old. Buildpacks provide a seamless mechanism for doing that *without Dockerfiles.* [Paketo Buildpacks](https://paketo.io) is a community-driven project that provides [Cloud Native Buildpacks](https://buildpacks.io) implementations for the most popular languages. Buildpacks provide great out-of-box support for many languages, but what if you need a buildpack that doesn’t exist yet? As a community-driven initiative, it may feel like the onus is on you to create that buildpack. Where would you even begin? 🤯
```